src/resmom/linux/alps.c

```diff
@@ -5306,7 +5306,14 @@ alps_create_reserve_request(job *pjob, basil_request_reserve_t **req)
 		 * that information into the ALPS reservation.
 		 * If not exclusive set it to shared by default
 		 */
```

Comment:
Can you change this comment? It is no longer set to shared by default.

Reply:
Oh good point @vstumpf.  I removed that line.  
@arungrover @vstumpf please take another look.